### DISCUSSION TITLE
Move folder / Merging folders - ignore.xml update

### DISCUSSION THREAD

**Discussion by Alister-Zeus** [hsoft/dupeguru#1038] [Q&A]
I'm on Windows and have a folders

e:\photos\illustrations\cartoons
e:\photos\illustrations\drawings
e:\photos\art\cartoons

I want to move them to

e:\illustrations\cartoons
e:\illustrations\drawings

can I do this then edit the ignore_list.xml replacing 

e:\photos\illustrations\cartoons -> e:\illustrations\cartoons
e:\photos\illustrations\drawings -> e:\illustrations\drawings
e:\photos\art\cartoons -> e:\illustrations\cartoons

Would I need to clear the cache?

**Comment by glubsy** [comment]
I believe you can yes, and I would also clear the cache although it probably won't matter much.
